src/test/resources/features/convertVideo/video.feature

```diff
@@ -0,0 +1,85 @@
+Feature: Video Controller
+
+  In order to convert a file
```

Comment:
Please consider improving this statement to something like: `In order to convert a video file`

Reply:
I modified it according your suggestion thanks.